docs/lab/08/index.md

```diff
@@ -0,0 +1,347 @@
+
+:::note
+In the above example, `192.168.0.0` is the *network address*, and `192.168.0.255` is the *broadcast address*. The first and last possible addresses in a subnet are reserved and should not be used for devices.
+:::
+
+Learn more about subnetting [here](https://networklessons.com/cisco/ccna-routing-switching-icnd1-100-105/what-is-subnetting).
+
+### Routers
+
+TODO
+
+### DHCP
+
+Dynamic Host Configuration Protocol (DHCP) is a network management protocol used by routers to dynamically allocate IP addresses to devices connected to its network. Routers dynamically assign addresses to new devices in the network, if they don't already have a *static* IP.
+
+## Wi-Fi
+
+**Wi-Fi** is a networking technology that allows devices to *wirelessly* communicate in a local network or interface the Internet. 
+
+Wi-Fi devices use radio waves to transmit data to one another in a network, through *wireless access points* (a wireless router, for instance). The access point (AP) connects directly to a local area network (LAN) and provides wireless connections for other devices in its proximity to use. 
+
+![wi-fi](images/Wi-Fi.gif)
+
+:::note
+Certain materials, such as stone with high metal content, can block Wi-Fi signals.
+:::
+
+### Protocol
+
+Wi-Fi uses the *802.11* IEEE standard, with different radio technologies:
+
+- 2.4 GHz radio band, ultra high frequency, (widely supported; Raspberry Pi Pico W uses this frequency)
+- 5 GHz radio band, super high frequency
+- others, not widely used
+
+Each radio frequency range is divided into a multitude of *channels*. Each device that uses a frequency band communicates over one channel. Ideally, each device is allocated a separate channel. Sometimes, though, when there are many devices using a Wi-Fi frequency band, devices might end up having to *share* a channel and wait their turn to transmit data, leading to lower communication speeds. 5 GHz provides more channels than 2.4 GHz, allowing more devices to use the band without interference. 
+
+### Wi-Fi on the Raspberry Pi Pico
+
+Apart from the RP2040 microcontroller, the Raspberry Pi Pico board also has a separate, on-board wireless interface. It is a 2.4 GHz Infineon chip, *CYW43439*. This chip deals only with Wi-Fi (and Bluetooth) communication, and it is connected to the RP2040 via SPI.
+
+![cyw43](images/cyw43.png)
+
+### Security
+
+#### Open Wi-Fi
+
+Open wireless networks don't require any authentication, and all transmissions are unencrypted. Any device that connects to an open network can easily intercept data sent in this network. This is unsecured and unsafe.
+
+#### Wired Equivalent Privacy (WEP)
+
+WEP is the first security protocol ever implemented. It uses data encryption based on generated key values. It was designed in 1197 and has since become obsolete, because it is really easy to crack and basically as unsafe as open Wi-Fi.
```

Comment:
```suggestion
WEP is the first security protocol ever implemented. It uses data encryption based on generated key values. It was designed in 1997 and has since become obsolete, because it is really easy to crack and basically as unsafe as open Wi-Fi.
```
No?

Reply:
Whoops